docs/utils.rst

```diff
@@ -4,65 +4,36 @@ DeepChem has a broad collection of utility functions. Many of these
 maybe be of independent interest to users since they deal with some
 tricky aspects of processing scientific datatypes.
 
-Array Utilities
----------------
-
-.. autofunction:: deepchem.utils.pad_array
-
-Data Directory
---------------
-The DeepChem data directory is where downloaded MoleculeNet datasets are stored.
-
-.. autofunction:: deepchem.utils.get_data_dir
-
-Print Threshold
----------------
-
-The printing threshold controls how many dataset elements are printed
```

Comment:
Could you copy over this comment about the print thresholds to the new section below on debugging utilities?